pages/linux/lvextend.md

```diff
@@ -0,0 +1,15 @@
+# lvextend
+
+> Increase the size of a logical volume.
+
+- Increase a volume's size to 120GB:
+
+`lvextend --size 120G {{logical volume}}`
```

Comment:
Perhaps this could be `logical_volume` instead of `logical volume` please? It would make it more consistent with other pages. Same goes for all the other examples.